### PR TITLE
More integration tests for endpoints with keycloak, updated vertx server

### PR DESCRIPTION
**Pull request by mlabuda**
+ minor fix of NPE in keycloak client

### CODE REVIEW

**Comment by ibuziuk** [comment]
[test]

File: src/main/java/io/fabric8/che/starter/client/keycloak/KeycloakClient.java [hunk 1]
```diff
@@ -45,6 +45,9 @@ public String getOpenShiftToken(String keycloakToken) throws JsonProcessingExcep
         ObjectMapper mapper = new ObjectMapper();
         JsonNode json = mapper.readTree(responseBody);
         JsonNode accessToken = json.get(ACCESS_TOKEN);
+        if (accessToken == null) {
```
Comment (ibuziuk) [inline src/main/java/io/fabric8/che/starter/client/keycloak/KeycloakClient.java]: @mlabuda Should now we create some exception for it - I do not like the idea of just returning null

Comment (mlabuda) [inline src/main/java/io/fabric8/che/starter/client/keycloak/KeycloakClient.java]: yeah. can you create a follow up issue on that? I will work on it on monday.

Comment (ibuziuk) [inline src/main/java/io/fabric8/che/starter/client/keycloak/KeycloakClient.java]: @mlabuda no problem - let's merge it in this shape and I will add exception myself ;-)

**Pull request merged**